backend/app/views/spree/admin/shared/_product_tabs.html.erb

```diff
@@ -20,5 +20,9 @@
     <%= content_tag :li, class: ('active' if current == :stock) do %>
       <%= link_to_with_icon 'home', Spree.t(:stock), stock_admin_product_url(@product) %>
     <% end if can?(:stock, Spree::Product) %>
+    <%= content_tag :li do %>
+      <%= link_to_with_icon 'eye-open', Spree.t(:see_in_store), product_path, :target => "_blank" %>
```

Comment:
Can you namespace the path as well? Probably call it `spree.product_path`. This way there's no conflicts for apps that might have a `main_app.product_path`, a `product_path` possibly defined in a helper, and the `spree.product_path`. Also, shouldn't an object get passed to the `product_path`? Like `spree.product_path @product`?